app/src/Talk/TalkController.php

```diff
@@ -475,15 +480,21 @@ public function unlinkSpeaker($eventSlug, $talkSlug, $username)
 
         $unlinkSpeakerUri = $talkUri . "/speakers/" . $userId;
 
-        try {
-            $talkApi->unlinkVerifiedSpeakerFromTalk($unlinkSpeakerUri);
-        } catch (Exception $e) {
-            $this->application->flash('error', $e->getMessage());
+        $isAdmin = $event->getCanEdit();
+        if (!$isAdmin && $talk->getStartDateTime()->format('U') > time()) {
```

Comment:
The check is made twice also (differnt sign), can a variable be used instead ? `$eventHasStarted = $talk->getStartDateTime()->format('U') > time()` then check is `!$isAdmin && $eventHasStarted` and the other check is `!isAdmin && !eventHasStarted`